.github/workflows/docker.yaml

```diff
@@ -49,8 +49,6 @@ jobs:
                       kimai/kimai2:${{ matrix.server }}-${{ matrix.type }}
                       kimai/kimai2:${{ matrix.server }}-${{ env.kimai_version }}-${{ matrix.type }}
                   push: true
-            - name: Test Lite
-              run: docker run --rm --entrypoint /assets/test-lite.sh kimai/kimai2:${{ matrix.server }}-${{ matrix.type }}
```

Comment:
Delete `assets/test-lite.sh` as well?

Reply:
With all the important changes of this PR this was something that I wanted to highlight later, but I agree that the test should not be removed.
It actually works and gives Kimai container some coverage, so no real reason to remove it (at least for me).

Reply:
That test ran `./bin/console kimai:version` and started apache/fpm. The test was run after the push so was pretty much useless other than to notify that something was wrong.

I'm happy to trust that the apache configs in the base images are tested upstream. I guess we could have hosed the apache config but it's not changed in 3 years.

Reply:
and that command is run during the build so the build will fail if it doesn't complete.

https://github.com/kimai/kimai/blob/main/Dockerfile#L291